### DISCUSSION TITLE
How Computational expensive is to open a real-time subscription?

### DISCUSSION THREAD

**Discussion by socramm9** [supabase/supabase#12287] [Questions] [answered]
I have an invitations table where a user can be invited to an organization.

I was thinking to use real-time to show the invitation to the user in the fronted. 
The thing ist, to receive an invitation will probably be a rare event so I was wondering if it would be not better to manually check for invitations

**Comment by thorwebdev** [comment, marked as answer]
Hey @socramm9, that sounds rare enough to probably not warrant the use of realtime. You can just fire off a SELECT query from your client when you need that information.

**Reply by socramm9** [reply]
I think I came up with an alternative. I will have a notifications table that is real-time and when this table recipes a invite notification I can run the select for my invites table
